### PR TITLE
Update development toolkit

### PR DESCRIPTION
**Pull request by MGatner**
* Updates Composer dependencies
* Adds GitHub Actions
* Adds PHPStan ~(not yet passing)~ passing!

### CODE REVIEW

File: src/FrequenciesTrait.php [hunk 1]
```diff
@@ -7,358 +7,388 @@
...
+	{
+		$min = $hour = 0;
+
+		if (! empty($time))
+		{
+			[
+				$min,
+				$hour,
+			] = $this->parseTime($time);
+		}
```
Comment (lonnieezell) [inline src/FrequenciesTrait.php:73]: I'll ignore it since it's automated - but this is really ugly :)

File: src/FrequenciesTrait.php [hunk 2]
```diff
@@ -7,358 +7,388 @@
...
+	 * @return $this
+	 */
+	public function yearly(string $time = null)
+	{
+		$min = $hour = 0;
+
+		if (! empty($time))
+		{
+			[
+				$min,
```
Comment (lonnieezell) [inline src/FrequenciesTrait.php]: Actually - this is really annoying and hard to read. Is there a way to turn this rule off?

Comment (MGatner) [reply src/FrequenciesTrait.php]: I don't let it run automatically bc I think it is too pushy. I will revert these ones.

**Comment by lonnieezell** [comment]
Thanks!

**Pull request merged**